src/gridfs/download.ts

```diff
@@ -1,14 +1,20 @@
+import type { ObjectId } from 'bson';
 import { Readable } from 'stream';
-import { AnyError, MongoDriverError, MongoInvalidArgumentError } from '../error';
 import type { Document } from '../bson';
+import type { Collection } from '../collection';
+import type { FindCursor } from '../cursor/find_cursor';
+import {
+  AnyError,
+  MongoDriverError,
+  MongoInvalidArgumentError,
+  MongoGridFSChunkError,
+  MongoGridFSStreamError
+} from '../error';
 import type { FindOptions } from '../operations/find';
```

Comment:
There's import changes here that aren't needed, I think you import sorting turned on in vscode? This is something we want to do as a lint fix down the line if you don't mind reverting these.